data/tabular/SIDER/meta.yaml

```diff
@@ -0,0 +1,220 @@
+---
+name: SIDER
+description: Database of marketed drugs and adverse drug reactions (ADR), grouped into 27 system organ classes.
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+targets:
+    - id: hepatobiliary_disorders
+      description: hepatobiliary disorders
+      type: boolean
+      names:
+          - noun: hepatobiliary disorders
+    - id: metabolism_and_nutrition_disorders
+      description: metabolism and nutrition disorders
+      type: boolean
+      names:
+          - noun: metabolism and nutrition disorders
+    - id: product_issues
+      description: product issues
```

Comment:
what does this mean? what is an example of a product issue?

Reply:
@AdrianM0 do you know what product issues is supposed to mean? Does this noun fit the column name?